apis/io-engine/protobuf/v1/snapshot-rebuild.proto

```diff
@@ -29,10 +43,14 @@ message CreateSnapshotRebuildRequest {
   bool                             resume = 3;
   // A bitmap where set bits indicate which
   // segment of the snapshot are allocated and therefore
-  // need to be transferred for the rebuild.
+  // need to be transferred from the source to rebuild.
   // If not specified the replica shall retrieve it
   // using nvme-admin command.
   optional SnapshotRebuildBitmap   bitmap = 4;
+  // Use the bitmap for the rebuild, or just walk the device range.
+  // If the bitmap is specified via the args use it, otherwise fetch
+  // it from the snapshot.
+  bool                         use_bitmap = 8;
```

Comment:
Why is this field numbered `8` ? There is no `7`.Also, I'm not sure of reason of numbering fields non-sequentially - does it help here in structure packing efficiently and reduce padding?

Reply:
It's for backwards compatibility, though since we never released I guess it's fine to start from 1-onwards